File: Retos/Reto #27 - CUENTA ATRÁS [Media]/python/AnzurezDev.py
import time

def cuenta_regresiva(comienzo: int, segundos:int):
    if isinstance(comienzo, int)>=0 and isinstance(segundos, int)>=0 :
        for i in range(comienzo, -1, -1):
            print(i)
            if i == 0 :
                break
            time.sleep(segundos)

cuenta_regresiva(5, 2)

File: Retos/Reto #27 - CUENTA ATRÁS [Media]/typescript/AnzurezDev.ts
const sleep = (seconds:number) => new Promise(resolve => setTimeout(resolve, seconds*1000));

async function cuenta_regresiva(inicio:number, segundos:number) {
    if (inicio>=0 && segundos>=0) {
        for (let index=inicio; index>=0; index--) {
            console.log(index);
            if (index==0) return;
            await sleep(segundos);
        }
    }
}

cuenta_regresiva(10, 1);